### ISSUE TITLE
process code about Reallm-Labs/android_control_test/

### ISSUE THREAD

**Issue by mengzchen** [Reallm-Labs/InfiGUI-R1#6]
Thank you for your excellent work!

I wonder how do you get [Reallm-Labs/android_control_test](https://huggingface.co/datasets/Reallm-Labs/android_control_test/)

Can you share the process code? I want to process the training part of android_control, and hope to re-use your process code for the test set.

Thanks!

**Comment by Sirius-L1** [comment]
Hi @mengzchen,

Thank you for your interest in our work!

The code for data processing is currently a bit disorganized and not yet ready for a public release. We might clean it up and release it in the future.

In the meantime, to facilitate your work, we have provided the processed training set which you can access directly here: [Reallm-Labs/android\_control\_train](https://huggingface.co/datasets/Reallm-Labs/android_control_train)

Please note that due to the large volume of images, we have not uploaded the image files to the repository. You will need to extract the images from the original dataset.

We will continue to update and release more open-source resources. Please stay tuned for our future updates :)

Best regards,

The InfiGUI-R1 Team
